src/Parser/Monk/Brewmaster/Modules/Features/Checklist.js

```diff
@@ -25,18 +26,19 @@ class Checklist extends CoreChecklist {
     castEfficiency: CastEfficiency,
     tp: TigerPalm,
     rjw: RushingJadeWind,
+    boc: BlackoutCombo,
   };
 
   rules = [
     new Rule({
       name: (
         <Wrapper>
-          Mitigate damage with <SpellLink id={SPELLS.IRONSKIN_BREW.id} />.
+          Mitigate damage with <SpellLink id={SPELLS.IRONSKIN_BREW.id} icon />.
```

Comment:
Adding all these icons shouldn't be necessary, icons should be shown by default now.